AndroidSDKFcm/src/main/java/com/leanplum/LeanplumPushFirebaseMessagingService.java

```diff
@@ -61,6 +61,9 @@ public void onNewToken(String token) {
   @Override
   public void onMessageReceived(RemoteMessage remoteMessage) {
     try {
+      if (Leanplum.getContext() == null) {
```

Comment:
Do we know why Leanplum.getContext() is null?  Otherwise with limited context on context this change looks good to me.

Reply:
Yes, we set the context when `start` call is issued, in case push notification is received and we are not started we can get in a situation where context is not set yet and/or `start` call not called